source/manage/health-check.rst

```diff
@@ -0,0 +1,34 @@
+Mattermost Health Check
+========================
+
+.. include:: ../_static/badges/ent-selfhosted.rst
+  :start-after: :nosearch:
+
+The Mattermost Health Check is a comprehensive evaluation of your system’s current operational health. It provides insights based on your system’s configuration, user activity, and log events, helping you maintain optimal performance and reliability.
+
+Why perform a Mattermost Health Check?
+--------------------------------------
+
+Regularly assessing the health of your Mattermost environment ensures that:
+
+- System performance remains optimal, with minimal delays or downtime.
+- User experience is consistently reliable, preventing missed messages or slow load times.
+- Security and compliance settings are up-to-date and effective.
+- Scalability aligns with your evolving usage patterns, supporting growth without issues.
+
+Perform a Mattermost Health Check
+----------------------------------
+
+Conducting a Mattermost Health Check is simple and involves just a few steps:
+
+1. :doc:`Generate a Support Packet </manage/generating-support-packet>`: The Mattermost Support Packet contains critical information about your Mattermost environment, including logs, configurations, and usage data.
+
+2. Submit Your Support Packet: Once you’ve generated the Support Packet, submit it through our Support System as a `standard support request <https://support.mattermost.com/hc/en-us/requests/new>`_. Please include “Health Check Provided” in the subject line. 
+
+3. Receive Your Health Check Report: Within 2 weeks of submitting your Support Packet, you will receive a comprehensive performance report. This report will include:
+
+- Detailed analysis of your current system health.
+- Actionable recommendations to enhance performance, reliability, and security.
+- Guidance on best practices for future system maintenance.
+
+If you have any questions or need assistance, please don't hesitate `contact our support team <https://mattermost.com/support/>`_, or visit the `Mattermost Help Center <https://support.mattermost.com/>`_.
```

Comment:
I wonder if we want to also make the point that contacting your Customer Success Manager or Technical Account Manager is an option here.

Raising this idea as it will let more readers know we offer this level of support.